src/test/java/uk/gov/pay/ledger/event/EventDaoITest.java

```diff
@@ -50,6 +53,60 @@ public void shouldInsertEvent() throws IOException {
         assertThat(objectMapper.readTree(result.get("event_data").toString()), is(objectMapper.readTree(event.getEventData())));
     }
 
+    @Test
+    public void shouldInsertEventIfDoesNotExistWhenInsertEventIfDoesNotExist() throws IOException {
+        Event event = anEventFixture()
+                .withEventDate(CREATED_AT)
+                .toEntity();
+        EventDao eventDao = rule.getJdbi().onDemand(EventDao.class);
+
+        var status = eventDao.insertEventIfDoesNotExistWithResourceTypeId(event);
```

Comment:
Can we use Object/data types ? It is not clear to understand what `var status` represents